pom.xml

```diff
@@ -113,6 +113,7 @@
         <module>org.eclipse.winery.highlevelrestapi</module>
         <module>org.eclipse.winery.tools.copybaragenerator</module>
         <module>org.eclipse.winery.model.substitution</module>
+        <module>org.eclipse.winery.model.threatmodeling</module>
```

Comment:
could we have this added to the artifacts included in the p2-maven-plugin a few lines below? Thanks

Reply:
added